integration/env/azure.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	DefaultAzureLocation             = "westeurope"
-	DefaultCommonDomainResourceGroup = "root_dns_zone_rg"
+	DefaultCommonDomainResourceGroup = "godsmack"
```

Comment:
This was a bug. The resource group containing the parent DNS zone is `godsmack`.